File: algorithm-programmers-stackqueue-valid-parentheses/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

class SolutionTest {
    @Test
    void trueCase() {
        assertTrue(new Solution().solution("()()"));
    }

    @Test
    void falseCase() {
        assertFalse(new Solution().solution("(())("));
    }
}

File: algorithm-programmers-stackqueue-valid-parentheses/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun trueCase() {
        assertTrue(Solution().solution("()()"))
    }

    @Test
    fun falseCase() {
        assertFalse(Solution().solution("(())("))
    }
}
